# Host-Virus Cophylogeny Trajectories: Investigating Molecular Relationships between Coronaviruses and Bat Hosts

## :memo: Background
<p align="justify">Bats, with their virus tolerance, social behaviors, and mobility, are reservoirs for emerging viruses, including coronaviruses (CoVs) known for genetic flexibility. Studying the cophylogenetic link between bats and CoVs provides vital insights into transmission dynamics, host adaptation, and the foundation for disease emergence and spread. In this study, we investigate the cophylogenetic patterns of 69 host-virus connections.
Prior research has yielded valuable insights into phenomena such as host switching, cospeciation, and other dynamics concerning the interaction between CoVs and bats. Nonetheless, a distinct gap exists in the current literature concerning a comparative cophylogenetic analysis focused on elucidating the contributions of sequence fragments to the coevolution between hosts and viruses.</p>

## :open_file_folder: Dataset

The [dataset]() includes sequences from 69 distinct CoV genotypes, which have their origins in 17 different bat species. These genetic sequences have
been obtained from GenBank and encompass the complete genome, polyprotein 1ab (ORF1ab), spike sequences of the CoVs, and the cytb gene sequences from the bat
specimens.
The 17 bat hosts include individuals in the "*Aselliscus stoliczkanus, Chaerephon plicatus, Hipposideros pratti, Miniopterus fuliginosus, Miniopterus magnate, Miniopterus pusillus, Myotis ricketti, Pipistrellus abramus, Rhinolophus affinis, Rhinolophus blasii, Rhinolophus ferrumequinum, Rhinolophus macrotis, Rhinolophus pearsoni, Rhinolophus pusillus, Rhinolophus sinicus, Tylonycteris pachypus*", and "*Vespertilio superans*". For molecular characterization, cytochrome b (cytb) gene sequences from all sampled bats were sourced from GenBank. This mitochondrial gene has proven instrumental in achieving species-level resolution for mammalian phylogenies within the Order.

## :pushpin: Results
<p align="justify">Among the 69 host-virus links examined, 47 showed significant cophylogeny based on ParaFit and PACo analyses, affirming strong associations. Focusing on two pivotal proteins, ORF1ab and spike, we conduct a comparative analysis of host and coronavirus phylogenies. 
For ORF1ab, specific window ranges in the multiple sequence alignment (positions 520-680, 770-870, 2930-3070, and 4910-5080) exhibit the lowest Robinson and Foulds RF distance (i.e. 84.62%), emphasizing their heigher contribution in the cophylogenetic association. Similarly, within the spike region, distinct window ranges (positions 0-140, 60-180, 100-410, 360-550, and 630-730) display the lowest RF distance at 88.46%.
Our analysis identifies six recombination regions within ORF1ab (positions 360-1390, 550-1610, 680-1680, 700-1710, 2060-3090, and 2130-3250), and four within the spike protein (positions 10-510, 50-560, 170-710, and 230-730). The convergence of minimal RF distance regions with combination regions robustly affirms the pivotal role of recombination in viral adaptation to host selection pressures.
Furthermore, horizontal gene transfer reveals prominent instances of partial gene transfer events, occurring not only among variants within the same host species but also crossing host species boundaries. This suggests a more intricate pattern of genetic exchange.</p>

## :dart: Conclusions
<p align="justify">
By employing a multifaceted approach, our comprehensive approach provides a nuanced understanding of the multifaceted interactions governing the coevolutionary dynamics between bat hosts and CoVs. This deeper insight informs our understanding of viral evolution and adaptation mechanisms, illuminating the broader dynamics driving viral diversity.
</p>

## ‚úîÔ∏è Citation
A manuscript for aPhyloGeo-pipeline is in preparation.

## üìß Contact
Please email us at: <Nadia.Tahiri@USherbrooke.ca> for any questions or feedback.
